app/models/author.rb

```diff
@@ -1,3 +1,5 @@
+require_dependency 'zine'
+require_dependency 'authorship'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.